va/va.h

```diff
@@ -1380,6 +1385,19 @@ typedef union _VAConfigAttribValContextPriority{
     uint32_t value;
 }VAConfigAttribValContextPriority;
 
+/** \brief Attribute value for VAConfigAttribLookAhead */
+typedef union _VAConfigAttribValLookAhead {
+    struct {
+        /** \brief look ahead is supported and is capable of returning CQM hint */
+        uint32_t lookahead_analysis_support    : 1;
+        /** \brief look ahead is supported and is capable of returning INTRA frame hint */
+        uint32_t lookahead_brc_support         : 1;
+        /** \brief reserved for future extension, must be zero */
```

Comment:
how about other variable from VALookAheadInfo, such as min_gop_size , does it also need a capability bit?